.github/ISSUE_TEMPLATE/Doc Report.yml

```diff
@@ -0,0 +1,61 @@
+name: Docs Report
+description: File a bug report
+title: "[Docs]: "
+labels: ["documentation"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this report!
+
+  - type: textarea
+    id: current-behaviour
+    attributes:
+      label: Current Behaviour
+      description: Tell us what went wrong?
+      placeholder: A bug happened!
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behaviour
+    attributes:
+      label: Expected Behaviour
+      description: Tell us what was supposed to happen?
+      placeholder: A bug should not have happened!
+
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
+      multiple: true
+      options:
+        - Firefox
+        - Chrome
+        - Safari
+        - Microsoft Edge
+        - Other
+
```

Comment:
```suggestion
```

Reply:
Why should these 2 be removed ?
The issue with docs could be browser based as well depending on the sphinx build

Reply:
Ahh well if it does fit in... then no worries 👍🏼